pkg/interactor/delete_data.go

```diff
@@ -8,8 +9,8 @@ func (d *DataInteractor) Delete(token, deviceID string) error {
 
 	err = d.DataStore.Delete(deviceID)
 	if err != nil {
-		d.logger.Error(err)
-		return err
+		d.logger.Error(ErrDeleteDataDB)
+		return ErrDeleteDataDB
```

Comment:
Hero too, I think you could wrap the error. Something like this:
`fmt.Errorf("error deleting data: %w", err)`

Reply:
Done!